src/lkl/posix-host.c

```diff
@@ -34,14 +34,16 @@
 
 #define NSEC_PER_SEC 1000000000L
 
-// The function used to implement the futex system call on top of lthreads
-int enclave_futex(
+// The functions used to implement the futex system call on top of lthreads
```

Comment:
Does this comment still hold?

Reply:
It does at the moment. I updated to reflect the refactoring change here.

Reply:
I can see how that would be misleading.

Reply:
Removed